WNPRC_Purchasing/src/org/labkey/wnprc_purchasing/table/WNPRC_PurchasingCustomizer.java

```diff
@@ -33,6 +33,23 @@ else if (matches(tableInfo, "ehr_purchasing", "purchasingReceiverOverview"))
             {
                 addRequestUpdateLinks((AbstractTableInfo) tableInfo);
             }
+            else if (matches(tableInfo, "ehr_billing", "aliases"))
+            {
+                addDisplayName((AbstractTableInfo) tableInfo);
+            }
+        }
+    }
+
+    private void addDisplayName(AbstractTableInfo aliasesTable)
+    {
+        String name = "displayName";
+        if (aliasesTable.getColumn(name) == null)
+        {
+            SQLFragment sql = new SQLFragment("(alias ||' - '|| groupName)");
```

Comment:
Will either alias or groupName ever be null?

Reply:
Condition now added